utils/putDataToIpdc.js

```diff
@@ -0,0 +1,39 @@
+import * as jsonld from 'jsonld';
+import { IPDC_JSON_ENDPOINT, IPDC_X_API_KEY } from '../env-config';
+export async function putDataToIpdc(subjectsAndData){  
+  //try catch so it doesnt messup other parts of the cron job
+  try {
+    let ttl=''
+    for(const subject of Object.keys(subjectsAndData)) {
+      const body=subjectsAndData[subject].body
+      ttl+=body;
+    }
+    
+    //bug/feature in jsonld rdf parser?
+    ttl=ttl.replaceAll(`"""`, `"`);
```

Comment:
it seems there is a bunch of chars that needs proper escaping. (I suspect it will break if you start adding new lines in textareas e.g.)
see https://github.com/digitalbazaar/jsonld.js/issues/44